resources/io/jenkins/infra/docker/Makefile

```diff
@@ -12,12 +12,10 @@ IMAGE_DEPLOY_NAME ?= $(IMAGE_NAME)
 IMAGE_PLATFORM ?= linux/amd64
 # Paths
 IMAGE_DOCKERFILE ?= $(IMAGE_DIR)/Dockerfile
-IMAGE_ARCHIVE ?= $(IMAGE_DIR)/image.tar
 HADOLINT_REPORT ?= $(IMAGE_DIR)/hadolint.json
 TEST_HARNESS ?= $(IMAGE_DIR)/cst.yml
 
 ## Locate command line tools. Defaults to rely on PATH
-CONTAINER_BIN ?= docker
```

Comment:
WDYT about either:
- Keeping this variable to keep the pattern of "any tool CLI can be overriden by end user through env var"
- Or also get rid of `HADOLINT_BIN` and `CST_BIN` (in that case we simplify the makefile code)

Reply:
Are there alternatives we will potentially use instead of hadolint & cst?
If there aren't, I'd simplify it to the max, as the previous version is still available in git if needed.

Reply:
Response: these tools could be called with docker if their binaries weren't present.

Reply:
Got rid of them.